### DISCUSSION TITLE
[Show]: Sites / Frameworks using Embla Carousel

### DISCUSSION THREAD

**Discussion by davidjerleke** [davidjerleke/embla-carousel#691] [Show and tell]
Are you using `embla-carousel` at your company/organization or is it part of your framework? Share the **URL** here!

**Sites/frameworks:**

- Mantine - [Link](https://mantine.dev/x/carousel/)
- Nuxt UI - [Link](https://ui3.nuxt.dev/components/carousel)
- Shadcn UI - [Link](https://ui.shadcn.com/docs/components/carousel)
- Shadcn Vue - [Link](https://www.shadcn-vue.com/docs/components/carousel.html)
- Shadcn Svelte - [Link](https://www.shadcn-svelte.com/docs/components/carousel)
- Shadcn Solid - [Link](https://shadcn-solid.vercel.app/docs/components/carousel)
- Muse UI - [Link](https://www.muse-ui.com/docs/components/carousel/)
- Keep React - [Link](https://react.keepdesign.io/docs/components/carousel)
- UI Layout - [Link](https://www.ui-layout.com/components/embla-carousel)
- Yamada UI [Link](https://yamada-ui.com/components/data-display/carousel)

**Comment by tobimori** [comment]
Some of the sites I built last year with embla:

https://sew-morlaix.com/
https://kazamagency.com/
https://www.enfore.com/us
https://factorycampus.de/
https://piqy.de/
https://www.kowa-leipzig.de/
https://pitchguru.com/

**Thanks for this really awesome library. It really made building sliders fun again.**

**Comment by multivoltage** [comment]
My company site:

https://www.conio.com/

**Comment by meirroth** [comment]
Done migrating to Embla Carousel on https://torahlive.com/
It's smooth and efficient ✨ Thanks for your great work!

**Reply by multivoltage** [reply]
From my Xiaomi slider in home with video does not work

**Reply by meirroth** [reply]
The main banner isn't using Embla (yet), but the other rows are. Thank you for the heads up!

**Comment by xnslx** [comment]
My demo store is one of them using embla.

[https://naixjewelry.vercel.app/](https://naixjewelry.vercel.app/)

**Reply by xnslx** [reply]
Here is my newly developed website, https://carousel-sanity.vercel.app/

Inspired by Zara, built with Remix, Embla and GSAP for seamless, fluid animations. This innovative carousel features a horizontal main slider with nested vertical sliders, this design offers a unique, dynamic way to showcase content. Built using Sanity CMS, it ensures a smooth user experience while maintaining flexibility and customizability.

**Reply by shyakadavis** [reply]
This is actually beautiful @xnslx 

Congrats. 🎉

**Reply by xnslx** [reply]
@shyakadavis Thank you for the kind words. I really appreciate it! 

-Xian

**Comment by sambedingfield** [comment]
Thank you for Embla! 🙏 It's been the perfect carousel library.

For my creative studio's website I use Embla for various carousels; timed, auto scrolling and auto playing:
https://wondermake.xyz/

**Comment by ggallo909** [comment]
Just released [bobsredmill.com](https://bobsredmill.com) using Embla for various components, from the mega menu cards to products showcase

**Reply by xnslx** [reply]
Hey I am a big fan of Bob’s products. So I take a look at the website. Is this what you are looking for? 


https://github.com/user-attachments/assets/f8bb865f-9a9c-4f66-97b3-a29bd89a69de

**Reply by ggallo909** [reply]
Thanks for reporting @xnslx there was a crossbrowsing issue that should be resolved now: the reveal effect for the red section in the footer for some browsers didn't work well with the z-index of the previous sections

**Reply by xnslx** [reply]
Quick response! Looks nice! Yeah, you might consider using `GSAP` to handle more complicated situations.
